src/man/author.rs

```diff
@@ -0,0 +1,6 @@
+/// An author entry.
+#[derive(Debug)]
```

Comment:
You can probably also derive Clone. Not necessary right now but you're not using Debug, too

Reply:
Good point!